packages/zowe-explorer/src/trees/uss/ZoweUSSNode.ts

```diff
@@ -153,6 +153,27 @@ export class ZoweUSSNode extends ZoweTreeNode implements IZoweUSSTreeNode {
         return ussEntry.attributes;
     }
 
+    public async fetchAttributes(): Promise<Types.FileAttributes> {
+        const ussEntry = UssFSProvider.instance.lookup(this.resourceUri, true) as UssFile | UssDirectory;
+        if (ussEntry == null || FsAbstractUtils.isDirectoryEntry(ussEntry)) {
+            return undefined;
+        }
+        const response = await this.getUssFiles(this.profile);
+        if (response.apiResponse?.length === 0) {
+            return undefined;
+        }
```

Comment:
`apiResponse` is an object here - the check on line 162 would always return `false`

Reply:
updated, thanks missed adding items in there 😄